core_1-14/src/main/scala/org/scalacheck/ops/GenOps.scala

```diff
@@ -0,0 +1,232 @@
+
+  /**
+   * Same as `Try(gen.randomOrThrow)`
+   */
+  @deprecated("The meaning of this method will change in the next major version. " +
+    "If you need a random sample on each call, you should use .randomOrThrow instead",
+    "2.3.0")
+  def tryGet: Try[T] = Try(gen.randomOrThrow)
+
+  /**
+   * An iterator made by continuously sampling the generator.
+   *
+   * @note If the generator has filters, then this method could return None a lot. If you want a
+   *       safe way to filter the Nones, see [[toIterator]]
+   *
+   * @return An iterator of Options which are None if the generator's filters ruled out the sample.
+   */
+  def sampleIterator: Iterator[Option[T]] = Iterator continually gen.sample
+
+  /**
+   * Converts this generator to an [[Iterator]] in the obvious (but potentially dangerous) way.
+   *
+   * @note This pulls an item from the generator one at a time, however, if the generator has too
+   * many restrictive filters, this can result in an infinite loop.
+   *
+   * @see [[toIterator]] for a safer, bounded alternative.
+   *
+   * @return An [[Iterator]] that returns only the defined samples.
+   */
+  def toUnboundedIterator: Iterator[T] = sampleIterator collect { case Some(x) => x }
+
+  /**
+   * Converts this generator into an infinite [[Iterator]] for a more flexible pull-based API.
+   *
+   * @param attempts the number of attempts to try generating a sample before throwing an execption
+   *
+   * @throws EmptyGenSampleException when samples cannot be generated after the given number of attempts
+   * @return An iterator that pulls from this generator or throws an exception
+   */
+  @deprecated("Use .iterator instead. " +
+    "If you need to specify retries, you can pass the argument GenConfig.defaults.withRetries(n) to .iterator",
+    "2.3.0")
+  def toIterator(attempts: Int): Iterator[T] = iterator(GenConfig.defaults.withRetries(attempts))
+
+  @deprecated("Use .iterator instead", "2.3.0")
+  def toIterator: Iterator[T] = iterator(GenConfig.defaults)
+
+  /**
+   * Converts this generator into a lazy [[Iterable]] so that it can be converted into a collection.
+   *
+   * @param attempts the number of attempts to try generating a sample before throwing an execption
+   *
+   * @throws EmptyGenSampleException when a sample cannot be generated after the given number of attempts
+   * @return A lazy iterable that pulls from this generator or throws an exception
+   */
+  @deprecated("Use .iterator.to() instead. " +
+    "If you need to specify retries, you can pass the argument GenConfig.defaults.withRetries(n) to .iterator",
+    "2.3.0")
+  def toIterable(attempts: Int): Iterable[T] = new Iterable[T] {
+    override val iterator: Iterator[T] = self.iterator(GenConfig.defaults.withRetries(attempts))
+  }
+
+  @deprecated("Use .iterator.to() instead", "2.3.0")
+  def toIterable: Iterable[T] = toIterable(GenConfig.defaults.retries)
```

Comment:
Most of this code is just copied from the shared source code, with the exception of this file.

I have:
- deprecated some methods that do not take an implicit `GenConfig` in favor of using `.iterator` which does take an implicit config
- made sure none of my changes break backwards binary compatibility